tests/FunctionalTest/ContainerAwareTest.php

```diff
@@ -27,6 +27,6 @@ public function testContainerAware()
         /** @var ContainerAwareController $class */
         $class = $container->get(ContainerAwareController::class);
 
-        $this->assertSame($container, $class->container);
+        $this::assertSame($container, $class->container);
```

Comment:
These code-style changes are unrelated to the PR, could you revert them?

Reply:
Assert method are static.